.github/release-please-config.json

```
@@ -2,6 +2,9 @@
   "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
   "last-release-sha": "2abd0ec82af934baa4296ed0f614f6c724c390fe",
   "packages": {
-    ".": { "package-name": "", "release-type": "node" }
+    "packages/catppuccin-unocss": {
+      "package-name": "@catppuccin/unocss",
```

This is just for PR names, I'd keep this empty
(See ctp/starlight for reference: https://github.com/catppuccin/starlight/blob/0b67ac8a758453cea86bc6cfa44af02a905cf680/.github/release-please-config.json#L7)

```suggestion
      "package-name": "",
```